extensions/answer_summarizers/models_test.py

```diff
@@ -50,19 +50,32 @@ def _create_answer_dicts_list(
             self, answers, times_spent_in_card, session_ids,
             classify_categories=(exp_domain.EXPLICIT_CLASSIFICATION,),
             num=None):
-        """This is similar to _create_answer_dict, except it provides a list
-        of N answers. It reuses answers, times_spent_in_card, and session_ids
-        cyclically to construct the list. When num isn't provided, the returned
-        list will be len(answers).
+        """This is similar to _create_answer_dict, except it simplifies building
+        several different answers at once.
+
+        Args:
+            answers: iterable of str. Each member is used cyclically to produce
```

Comment:
In general, answers may not be strings. Can we make the type of this list(*) instead (and actually pass in lists)?

In general, better for the next few args to be list(float), list(str), list(str) too -- more precise.

Reply:
Done.